src/components/LatestChanges/index.js

```diff
@@ -1,171 +1,135 @@
+import Image from 'next/image'
 import BackButton from '../Layout/BackButton'
+import closeWorkOrder from '../../../public/images/closeworkorder.jpg'
+import detailsOfFurtherWork from '../../../public/images/detailsoffurtherwork.jpg'
+import additionalNotes from '../../../public/images/additionalnotes.jpg'
```

Comment:
Does this grab the file path, or the binary itself?

Reply:
As far as I'm aware it grabs the file path